Design/Workbench/design-development.md

```diff
@@ -0,0 +1,22 @@
+# Development
+
+## Coding Style
+
+It is proposed we introduce a new Python style guide for Mantid. All new and refactored code should then follow this guide. The [IPython style guide][ipython-style] offers quite nice guidelines and it is suggested that we
+follow these guidelines where applicable, including a template file for new code that could be used in the `class_maker` to generate new files.
+
+## Static Analysis
+
+We will no longer be able to rely on a compiler to catch simply errors - it is therefore imperative that we use static analysis to its fullest extent. While `flake8` is good it is not as thorough as `pylint` and given our
```

Comment:
Run `pylint` against `master` and be quickly fix any found issues. `flake8` is more appropriate for pull requests because it gives consistent results.